subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -68,7 +70,10 @@ static void tx_lll_flush(void *param);
 static memq_link_t link_lll_prepare;
 static struct mayfly mfy_lll_prepare = {0U, 0U, &link_lll_prepare, NULL, NULL};
 
-static struct ll_adv_iso_set ll_adv_iso[BT_CTLR_ADV_SET];
+static struct ll_adv_iso_set ll_adv_iso[CONFIG_BT_CTLR_ADV_ISO_SET];
+static struct lll_adv_iso_stream
+			stream_pool[CONFIG_BT_CTLR_ADV_ISO_STREAM_COUNT];
+static void *stream_free;
```

Comment:
I assume that `stream_free` is a pointer to an index in `stream_pool`: Shouldn't the type be this then? 
```suggestion
static lll_adv_iso_stream *stream_free;
```